troubleshooting.md

```diff
@@ -2,6 +2,12 @@
 
 ## MacOS 
 
+First thing to check is that you are running the latest version of Hugo and that it's the Extended version. You can see the list of Hugo releases [here](https://github.com/gohugoio/hugo/releases). 
+
+```shell
+hugo version
+```  
+
```

Comment:
This isn't endemic to macOS! Checking for the latest version is a great best practice for all platforms and has solved issues for me on Ubuntu in the past. Can we move to a generic, non-macOS section?

Also - we should maybe give an example output? (mine is below). This way folks know that it's not easy to pick out the latest version (currently `0.108.0`, which I'm running) and how to see if it's extended (string `extended`) appears in output.

```
hugo version
hugo v0.108.0-a0d64a46e36dd2f503bfd5ba1a5807b900df231d+extended linux/amd64 BuildDate=2022-12-06T13:37:56Z VendorInfo=snap:0.108.0
```